README.mediawiki

```diff
@@ -1210,7 +1210,14 @@ Those proposing changes should consider that ultimately consent may rest with th
 | DNS Payment Instructions
 | Matt Corallo, Bastien Teinturier
 | Standard
-| Proposed
+| Draft
```

Comment:
Please don't change the status of other entries (CI is failing for this reason).

Reply:
Good catch, sorry about that